specs/utility/time.table

```diff
@@ -19,5 +19,8 @@ schema([
         aliases=["date_time"]),
     Column("iso_8601", TEXT, "Current time (ISO format) in the system"),
 ])
+extended_schema(WINDOWS, [
+    Column("timestamp_sys100ns", BIGINT, "Timestamp value in 100 nanosecond units."),
```

Comment:
Instead perhaps `win_timestamp`? This value is pretty unique to Windows, and I think the `sys100ns` information could be derived from the table description.